.idea/vcs.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you add this to .gitignore?

Reply:
`.idea` is in the gitignore I think that should cover it?

Reply:
ok